airflow/migrations/versions/e3a246e0dc1_current_schema.py

```diff
@@ -220,6 +220,18 @@ def upgrade():
             sa.PrimaryKeyConstraint('id'),
         )
 
+    if 'task_notes' not in tables:
+        op.create_table(
+            'task_notes',
+            sa.Column('task_id', sa.String(length=250, **COLLATION_ARGS), nullable=False),
+            sa.Column('dag_id', sa.String(length=250, **COLLATION_ARGS), nullable=False),
+            sa.Column('execution_date', sa.DateTime(), nullable=False),
+            sa.Column('timestamp', sa.DateTime(), nullable=False),
+            sa.Column('user_name', sa.String(length=250), nullable=True),
+            sa.Column('task_note', sa.Text(), nullable=True),
+            sa.PrimaryKeyConstraint('task_id', 'dag_id', 'execution_date', 'timestamp'),
+        )
+
```

Comment:
You will need to create a new migration instead of editing the existing one -- when upgrading existing installs old migrations are not re-run.

Reply:
Yes, I was not aware. I'll fix it 😄